MapboxNavigation/NavigationViewController.swift

```diff
@@ -147,13 +147,30 @@ open class NavigationViewController: UIViewController, NavigationStatusPresenter
      */
     public var shouldManageApplicationIdleTimer = true
     
+    /**
+     Enables highlighting of the destination building when arriving.
+     */
+    public var highlightDestinationBuildings: Bool = false {
+        didSet {
+            mapView?.buildingHighlightingEnabled = highlightDestinationBuildings
+        }
+    }
+    
+    /**
+     When destination building highlighting is enabled, this option controls whether the building will be presented in 3d or 2d.
+     
+     The default value of this property is `true`.
+     */
+    public var highlightBuildingsIn3D: Bool = true
+    
     var isConnectedToCarPlay: Bool {
         if #available(iOS 12.0, *) {
             return CarPlayManager.isConnected
         } else {
             return false
         }
     }
+    var buildingHighlightCoordinates: [CLLocationCoordinate2D] = []
```

Comment:
I don't see that `buildingHighlightCoordinates` is used anywhere, do we really need it?